octobot_trading/exchanges/util/exchange_util.py

```diff
@@ -135,28 +135,42 @@ def _get_minimal_exchange_config(exchange_name, exchange_config):
     }
 
 
-async def is_compatible_account(exchange_name: str, exchange_config: dict, tentacles_setup_config) -> (bool, str):
+async def is_compatible_account(exchange_name: str, exchange_config: dict, tentacles_setup_config, is_sandboxed: bool) \
+        -> (bool, bool, str):
+    """
+    Returns details regarding the compatibility of the account given in parameters
+    :return: (True if compatible, True if successful login, error explanation if any)
+    """
+    exchange_type = exchange_config.get(common_constants.CONFIG_EXCHANGE_TYPE, common_constants.DEFAULT_EXCHANGE_TYPE)
+    # non spot exchange config then check supports
     local_exchange_manager = exchange_manager.ExchangeManager(
         _get_minimal_exchange_config(exchange_name, exchange_config),
         exchange_name
     )
     local_exchange_manager.tentacles_setup_config = tentacles_setup_config
     # Avoid exchange authentication error log
-    local_exchange_manager.is_collecting = True
+    local_exchange_manager.check_credentials = False
+    local_exchange_manager.is_sandboxed = is_sandboxed
     await exchange_factory.search_and_create_spot_exchange(local_exchange_manager)
```

Comment:
spot exchange? Is it working when using future exchange?

Reply:
Right, for now it works as is but we should not keep it, I changed it